src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -580,16 +580,23 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
             }
         }
 
-        private static void ReportParameterErrors(
-            Symbol owner,
+
+        public static void ReportParameterErrors(
+            Symbol? owner,
             BaseParameterSyntax parameterSyntax,
-            ParameterSymbol parameter,
+            int ordinal,
+            bool isParams,
+            TypeWithAnnotations paramTypeWithAnnotations,
+            TypeSymbol paramType,
```

Comment:
>TypeSymbol paramType [](http://example.com/codeflow?start=12&length=20)

Consider removing this parameter and replacing with `paramTypeWithAnnotations.Type`.